src/pages/Ingame/index.js

```diff
@@ -1,35 +1,45 @@
 import React, { useState, useEffect } from "react";
 import { useLocation, useHistory } from "react-router-dom";
 import styled from "styled-components";
-import GameItem from "../../components/GameItem";
+import GameItem from "../../components/Ingame/GameItem";
+import GameCard from "../../components/Ingame/GameCard";
 import RuleBook from "../../components/RuleBook";
 import Modal from "../../components/Modal";
-import { emitJoinTeam, socket } from "../../utils/socket";
-import { getMiddleResult, getCurrentScore, sumScores, sumResults, makeData } from "../../utils/game";
+import {
+  socket,
+  emitJoinTeam,
+  getMiddleResult,
+  getCurrentScore,
+  sumScores,
+  sumResults,
+  makeData,
+  checkSpecialRound,
+} from "../../utils";
 
 export default function Game() {
   const history = useHistory();
   const location = useLocation();
   const teamInfo = location.pathname.split(":")[1].split("-");
   const [channelId, roomId, team] = teamInfo.slice(0, 3);
```

Comment:
이 부분 궁금했던건대 구지 3개로 슬라이스 한 이유가 있을까요? 
요런식으로 하면 깔끔할 거 같은데 구조분해 하면서 형변환이 바로 가능한 방법이 있는지 한번 알아봅시다
이게 되는 지는 실험을 안해봐서. 배열일 경우 구조분해 할당하면서 바로 값이 할당도 가능합니다.
```suggestion
  const [channelId, roomId, team, round=ParsInt(round,10)] = teamInfo;
```

Reply:
round는 const가 아니어서 이렇게 했습니다!

Reply:
이해했습니다!